code/modules/mob/living/simple_animal/friendly/lizard.dm

```diff
@@ -24,13 +24,63 @@
 	can_collar = TRUE
 	mob_biotypes = MOB_ORGANIC | MOB_BEAST | MOB_REPTILE
 	gold_core_spawnable = FRIENDLY_SPAWN
+	var/eating_sound = 'sound/weapons/bite.ogg'
+	/// Lizards start with a tail
+	var/has_tail = TRUE 
+
+/mob/living/simple_animal/lizard/handle_automated_action()
+	if(!stat && !buckled)
+		if(prob(1))
+			custom_emote(EMOTE_VISIBLE, pick("sticks out its tongue", "wags its tail.", "lies down."))
+
+	if(isturf(loc) && !incapacitated())
+		for(var/mob/living/M in view(1, src))
+			if(!M.stat && Adjacent(M) && HAS_TRAIT(M, TRAIT_EDIBLE_BUG))
+				custom_emote(EMOTE_VISIBLE, "eats \the [M]!")
+				playsound(loc, eating_sound, 20, 1)
+				M.death()
+				break
+		for(var/obj/structure/spider/spiderling/Spider in view(1, src))
+			if(Adjacent(Spider) && HAS_TRAIT(Spider, TRAIT_EDIBLE_BUG))
+				if(prob(90)) // Slippery things aren't they?
+					visible_message("<span class='notice'>The spiderling skitters away from the [src]!")
+					Spider.random_skitter()
+					return
+				else
+					custom_emote(EMOTE_VISIBLE, "eats \the [Spider]!")
+					playsound(loc, eating_sound, 20, 1)
+					qdel(Spider)
+					break
+
+/mob/living/simple_animal/lizard/verb/lose_tail()
+	set name = "Lose tail"
+	set desc = "Allows you to lose your tail and escape a sticky situation. Takes 5 minutes for your tail to grow back."
+	set category = "Animal"
+
+	if(stat != CONSCIOUS)
+		return
+	if(!has_tail)
+		to_chat(usr, "<span class='warning'>You have no tail to shed!</span>")
+		return
+	for(var/obj/item/grab/G in usr.grabbed_by)
+		var/mob/living/carbon/M = G.assailant
+		usr.visible_message("<span class='warning'>[usr] suddenly sheds their tail and slips out of [M]'s grasp!</span>")
+		M.KnockDown(3 SECONDS) // FUCK I TRIPPED
```

Comment:
The 3 second knockdown is kind of questionable but I get that it's to give the lizard time to escape.

Reply:
This should be fine, like 6 seconds of total knockdown on a 5-minute timer isn't very useful, it's just a small thing to make them more distinct